histonets/cli.py

```diff
@@ -1,33 +1,34 @@
 # -*- coding: utf-8 -*-
 import click
 
-from .utils import open_image
+from .utils import io_handler
 from .histonets import adjust_contrast, adjust_brightness
 
 
 @click.group()
+@click.version_option()
 def main():
     """Histonets computer vision application for image processing"""
```

Comment:
I'm not sure where the right place for it is, but comments with the proper form of using the cli, i.e., which options to pass in, would be nice attached to this commit.